.github/workflows/publish-chart.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Chart
+
+on:
+  workflow_call:
+    inputs:
+      release_version:
```

Comment:
Does the version have to follow semver? If so adding basic regex validation would be good.

Reply:
added via `loft-sh/github-actions/.github/actions/semver-validation@semver-validation/v1`